source/solvers/postprocessing_velocities.cc

```diff
@@ -40,14 +40,35 @@ AverageVelocities<dim, VectorType, DofsType>::calculate_average_velocities(
   // Get the inverse of the total time with the first time step since
   // the weighted velocities are calculated with the first velocity when
   // total time = 0.
-  inv_range_time = 1. / ((current_time - real_initial_time) + dt_0);
+  total_time_for_average = (current_time - real_initial_time) + dt_0;
+  inv_range_time         = 1. / total_time_for_average;
 
   // Calculate the average velocities.
   average_velocities.equ(inv_range_time, sum_velocity_dt);
```

Comment:
what is this `equ` method?

Reply:
https://www.dealii.org/current/doxygen/deal.II/group__TrilinosWrappers.html#gaa17a3b93459cf0c884354875debd3797